mimic/test/test_identity_objects.py

```diff
@@ -56,6 +297,46 @@ def test_listing_templates(self):
         eeapi.remove_template(eeapi_template_id)
         self.assertEqual(len(eeapi.list_templates()), 0)
 
+    def test_listing_templates_tenant(self):
+        tenant_id = 'some-tenant'
```

Comment:
I thought I had caught all of that...will review and update. Would be nice if the `tox -e docs` caught this since it does catch that on everything else.